### DISCUSSION TITLE
如何修改群验证消息？

### DISCUSSION THREAD

**Discussion by Sjxixi-y** [simple-robot/Discussions#167] [Q&A]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

如何修改：加群方式 --  需要回答问题并管理员审核中的问题？这些更详细的消息在API中的那个部分。

**Comment by ForteScarlet** [comment]
mirai没有提供这方面的能力，自己登录群主账号手动修改吧
